modules/configuration/pom.xml

```diff
@@ -70,6 +70,11 @@
         </dependency>
 
         <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-extended-api</artifactId>
```

Comment:
What is this dependency for?

Reply:
This is for `ExtendedTableConfigurationSchema`.